src/Pages/Help/Readme.elm

```diff
@@ -58,16 +62,142 @@ subscriptions model =
+
+When you are in a given mode, actions affect that player, so if you click Play on a track when in Local mode, it will 
+
+play through the browser, likewise, when in Kodi mode all commands are sent to Kodi. You can also add media to 
+
+other playlists by clicking the menu buttons (three dots vertical) on most media items.
+
+### Video streaming
+
+Video streaming via HTML5 "sort of" works, it really depends on the codec used. An embedded VLC player is also 
+
+available with better codec support. This looks like the best we can get until Kodi supports transcoding. **Chrome** 
+
+**users: ** Chrome has removed support for vlc/divx plugins so streaming a video requires a [Chrome friendly codec]. 
+
+For best results use Chrome with mp4 video that has 2 channel audio (5.1 audio doesn't seem to work).
+
+## Kodi settings via the web interface
+
+You can change most of the settings you would find in Kodi via the settings page in the web interface. Some 
+
+settings have been omitted as they require interaction with the GUI and others are just a basic text field with no 
+
+options.
+
+## Contributing
+
+If you would like to make this project better I would appreciate any help. You can do pull requests against the 
+
+main branch. See the [developers] section for information about getting a dev environment up and running the project.
+
+### Translations
+
+At the moment, there are a handful of languages available but more can be easily added. More strings are always being
+
+added so always consider english as the source of truth. So if you see something in english but want it in your language, 
+
+I need you! To contribute, send us a Pull request or if you don't know git, a link to the language file.
+
+_English is the only real complete translation file so start with that as your base_.
+
+## Donate
+
+Are you a fan of Chorus? You can buy Alex a beer to say thanks :)
+
+## License
+
+This program is free software; you can redistribute it and/or modify it under the terms of the GNU General Public License 
+
+as published by the Free Software Foundation; either version 2 of the License, or (at your option) any later 
+
+version.
+
+This program is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY; without even the 
+
+implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU General Public 
+
+License for more details.
+
+You should have received a copy of the GNU General Public License [along with this program]; if not, write to the 
+
+Free Software Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA 02110-1301 USA.
+
+[Click here for more information] .
+
+[Click here for more information]: https://github.com/xbmc/chorus2/blob/master/src/lang/en/license.md
+[along with this program]: https://github.com/xbmc/chorus2/blob/master/LICENSE
+[developers]: http://localhost:1234/help/developers
```

Comment:
We can't have links to localhost, they would need to be relative. So something like `/help/developers` would need to work here